docs/modules/operation/pages/quick-start/notification-config.adoc

```diff
@@ -0,0 +1,64 @@
+
+= Step 8: Configure Notifications
+
+This section describes how to configure and customize your {page-component-title} notifications.
+
+NOTE: This step is separate from xref:operation:quick-start/notifications.adoc[setting up default notifications].
+We assume that you have followed the previous steps and are ready to proceed to more in-depth work with notifications.
+
+== Configuration and customization
+
+Now that you have xref:operation:quick-start/baseline.adoc[established your environment's baseline], it's time to take your notes and configure your custom notifications.
+Establishing notifications that are tailored for your environment and monitoring requirements lets the right people see and respond to the right issues in a timely manner.
+
+Here, we will describe how to configure event notifications and path outages.
+Event notifications can be configured to provide UEI-specific coverage, allowing for granular control of incoming information.
+Path outages prevent the generation of excess notifications by identifying network paths where failures may cause multiple nodes to appear offline.
+
+== Configure event notifications
+
+An event notification defines a destination path and message template for a specific event UEI.
+It is possible to define multiple notifications for the same UEI to use different filters, destination paths, and message templates.
+Follow these steps to configure event notifications:
+
+. Click the *Gear* symbol in the top-right of the screen.
+. Under Event Management, click menu:Configure Notifications[Configure Event Notifications].
+A list of events that are configured for notifications is displayed.
+. Click *Add New Event Notification*, choose the UEI that will trigger a notification, and click *Next*.
+** (Optional) Build a rule to match a subset of IP addresses or services, and click menu:Validate Rule Results[Next] or *Skip Results Validation*.
+. In the *Choose Path* screen, specify information to include in the notification, including the destination path, subject (for email messages), body text, and so on.
+. Click *Finish*.
+
+NOTE: Event notification configuration is stored in `$\{OPENNMS_HOME}/etc/notifications.xml`.
+Changes to this file happen immediately, with no need to restart {page-component-title}.
+
+=== Filter rules
+
+A filter rule lets you define which notifications to send or save. Strict rules are always evaluated, and if there is no node or interface associated with the event, the notification will not be saved. 
+
+By default, the filter rule configured in `$\{OPENNMS_HOME}/etc/notifications.xml` (for example, `<rule>IPADDR != '0.0.0.0'</rule>`) is not strict.
+That means that events with no associated node or interface will not validate the rule, and the notification would be saved.
+That means if there is an event that is not associated with a node or interface, it will not validate the rule and, by default, the notification would be saved.
```

Comment:
```suggestion
By default, the filter rule configured in `$\{OPENNMS_HOME}/etc/notifications.xml` (for example, `<rule>IPADDR != '0.0.0.0'</rule>`) does not have the `strict` flag enabled.
That means if there is an event that is not associated with a node or interface (and therefore no `IPADDR` value), it will bypass validation and the notification would be processed.
```